packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,355 @@
+}
+
+@media (min-width:25rem) {
+  .emotion-6 {
+    background-size: 77px 22px;
+  }
+}
+
+@media (min-width:63rem) {
+  .emotion-6 {
+    background-size: 93px 27px;
+  }
+}
+
+.emotion-22 {
+  padding: 0.5rem;
+  display: inline-block;
+  max-width: calc(100% - 96px);
+}
+
+@media (min-width:25rem) {
+  .emotion-22 {
+    padding: 0.5rem 1rem;
+    max-width: calc(100% - 117px);
+  }
+}
+
+@media (min-width:37.5rem) {
+  .emotion-22 {
+    padding: 0.5rem 1rem;
+    max-width: calc(100% - 178px);
+  }
+}
+
+@media (min-width:63rem) {
+  .emotion-22 {
+    padding: 1rem;
+    max-width: initial;
+  }
+}
+
+.emotion-14 {
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSerif,Helvetica,Arial,sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+  color: #222222;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .emotion-14 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .emotion-14 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+.emotion-12 {
+  color: unset;
+}
+
+.emotion-12:before {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  content: '';
+  overflow: hidden;
+  z-index: 1;
+}
+
+.emotion-12:visited .podcast-promo__visited {
+  color: #AEAEB5;
+}
+
+.emotion-12:focus .podcast-promo__focus {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+}
+
+.emotion-10 {
+  -webkit-clip-path: inset(100%);
+  clip-path: inset(100%);
+  -webkit-clip: rect(1px,1px,1px,1px);
+  clip: rect(1px,1px,1px,1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+  margin: 0;
+}
+
+.emotion-16 {
+  font-size: 0.9375rem;
+  line-height: 1.125rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  max-width: 30rem;
+  color: #6E6E73;
+  margin-top: 0.5rem;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .emotion-16 {
+    font-size: 0.9375rem;
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .emotion-16 {
+    font-size: 0.875rem;
+    line-height: 1.125rem;
+  }
+}
+
+.emotion-18 {
+  width: 12px;
+  color: #6E6E73;
+  fill: currentColor;
+  position: relative;
+  top: 2px;
+  margin-right: 8px;
+}
+
+.emotion-20 {
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  color: #6E6E73;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .emotion-20 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .emotion-20 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+<div>
+  <section
+    class="emotion-26 emotion-27"
+    role="region"
+  >
+    <div
+      class="emotion-4 emotion-5"
+    >
+      <svg
+        class=" emotion-0 emotion-1"
```

Comment:
I don't see the tabindex requirement for ie11

Reply:
👍  is that requirement asking for a tabIndex of `-1`?  I added that - no other SVGs in psammead seem to have it though

Reply:
yeh -1. It was a requirement from Catharine. It must be just in this context then 🤔

Reply:
Had a deeper look - seems the `mediaIcons` in psammead are using this `focusable` attribute: https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-assets/src/svgs/mediaIcons.jsx#L10

Seems to be a non-standard IE attribute that achieves the same thing as tab index of -1 so will inherit that when I moved the icons in there